WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert Step Item to Angular

## Changes

Converted Step Item to Angular.

## Test

Make sure the Grade By Team view for a specific workgroup works properly.
1. Open the Classroom Monitor for a run
2. Click on the Grade By Team icon in the left side bar
3. Click on a workgroup
4. Click on a step. The step should expand and display the workgroup's work and the score and comment section.
5. Click "Expand All". All the steps should expand.
6. Click "Collapse All". All the steps should collapse.

Closes #768
